pkg/types/status.go

```diff
@@ -20,18 +23,21 @@ var (
 		"will_not_fix",
 		"fix_deferred",
 		"end_of_life",
+		"needed",
+		"pending",
```

Comment:
Does it make sense to simply normalise these cases to `fix_deferred`? These all seem to imply the fix is not available _yet_ but will be available in the _future_, as opposed to `will_not_fix`?